### PR TITLE
Update flower and Mako in supported images

### PR DESCRIPTION
**Pull request by ashb**
Neither CVE affects us, but we should keep up to date

**What this PR does / why we need it**:

**Special notes for your reviewer**:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] If a new distribution or new version of Airflow is added, please make sure:
  + [ ] to follow all of the directions in the `README.md`
  + [ ] that there are Dockerfiles in all base image directories
  + [ ] that all of the new or modified Dockerfiles build successfully
- [ ] If changing an existing image, please add an applicable test in `.circleci/bin/test-airflow-image.py`

### CODE REVIEW

**Comment by jedcunningham** [comment]
Also need to bump 1.10.15 and 2.1.4 to the next dev versions too.

**Review by kaxil — APPROVED** [review]

**Comment by kaxil** [comment]
> Oh ugh.
> 
> ```
>     apache-airflow-providers-celery 1!2.0.0 depends on flower<1.0 and >=0.7.3
>     The user requested (constraint) flower==1.2.0
> ```
> 
> Why does celery extra even depend on flower >_<

argh..

**Comment by ashb** [comment]
```
    apache-airflow-providers-celery 1!2.1.0 depends on celery>=5.1.2 and ~=5.1
    The user requested (constraint) celery==4.4.7
```
Chasing the deps will continue until morale improves

File: 2.1.4/buster/build-time-pip-constraints.txt [hunk 1]
```diff
@@ -163,7 +163,7 @@ cached-property==1.5.2
 cachetools==4.2.2
 cassandra-driver==3.25.0
 cattrs==1.5.0
-celery==4.4.7
+celery==5.2.3
```
Comment (jedcunningham) [inline 2.1.4/buster/build-time-pip-constraints.txt:166]: We'd need to backport this too:
https://github.com/apache/airflow/pull/17397

And part of why we [ignore the celery upgrade already](https://github.com/astronomer/ap-airflow/blob/b9729541a9ec478e0bc975bb91c7a4d7e715fddf/2.1.4/buster/trivyignore#L12).

Comment (ashb) [reply 2.1.4/buster/build-time-pip-constraints.txt:166]: Yeah I was worried that might happen.

Repackaging the provider might be easier?

Comment (ashb) [reply 2.1.4/buster/build-time-pip-constraints.txt:166]: Yeah I was worried that might happen.

Repackaging the provider might be easier?

**Pull request closed without merging**